src/components/counter/containers/CounterContainer.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from "react";
+
+import CounterView from "../components/CounterView/CounterView";
+
+class CounterContainer extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      countValue: 0,
+      isEven: true,
+    };
+  }
+
+  handleIncrement = () => {
+    this.setState((state) => ({
+      countValue: state.countValue + 1,
+      isEven: !state.isEven,
```

Comment:
you need to recalculate parity, not just toggle the flag. If I add button with +2 your code will not work correctly